project/src/main/webapp/emailParse.js

```diff
@@ -52,44 +41,143 @@ class MLModelEmailParse {
                 console.log("nonEmpty", nonEmpty);
                 console.log("highest confidence", orderedConfidence);
                 answer.innerHTML = mlDictAnswer["answer"];
+                answer.value = mlDictAnswer["answer"];
+                initMap();
                 console.timeEnd("Using Promises Test");
             }
         });
     }
 
-    extractEmailBodiesToArray(exampleDict) {
+    displayEmailBodies(nonEmpty) {
```

Comment:
I thought Brandon already merged all this. Can you check with him and make sure you have the most up-to-date version from master?

Reply:
yeah, sorry! Lemme get rid of these files!